src/popup/components/Scrollable.vue

```diff
@@ -7,7 +7,7 @@
 </template>
 
 <style lang="scss" scoped>
-@use '../../styles/variables';
+@use '../../styles/variables' as *;
```

Comment:
If the whole project we use variables as `variable.$color-some-color` should we diverge from it in favour of using just a variable name?

Reply:
We are not using any 3rd party frameworks so in my opinion it's easier to not use the `viariables` prefix. We can gradually migrate into this approach.